trin-core/src/types/accumulator.rs

```diff
@@ -47,6 +49,15 @@ impl MasterAccumulator {
         Self::default()
     }
 
+    /// Load default trusted master acc
+    pub fn try_from_file(master_acc_path: PathBuf) -> anyhow::Result<MasterAccumulator> {
+        let mut path = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
+        path.push(master_acc_path);
```

Comment:
`CARGO_MANIFEST_DIR` is a defined environment variable, set by cargo, which will always give us a path. Now, maybe `master_acc_path` is an invalid path... but that would cause an error on line 56. I don't see a side effect in this case that occurs before line 56 - where it might have trouble reading an invalid path, in which case we return an error.